src/test/javascript/spec/component/shared/metis/posting-content/posting-content.component.spec.ts

```diff
@@ -492,6 +492,22 @@ describe('PostingContentComponent', () => {
             ]);
         }));
 
+        it('should compute parts when referencing a faq', () => {
+            component.content = `I want to reference [faq]faq(/courses/1/faq?faqId=45)[/faq].`;
+            const matches = component.getPatternMatches();
+            component.computePostingContentParts(matches);
+            expect(component.postingContentParts).toEqual([
```

Comment:
This is now a signal

```suggestion
            expect(component.postingContentParts()).toEqual([
```